src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
 import NextLink from 'next/link';
+import { useNumericRouteParams } from 'core/hooks';
 import { useRouter } from 'next/router';
 import { useState } from 'react';
-import { useTheme } from '@mui/material/styles';
 import {
-  AppBar,
+  Avatar,
   Box,
-  Drawer,
+  Divider,
   IconButton,
   List,
   ListItem,
-  Toolbar,
+  ListItemIcon,
+  ListItemText,
+  Typography,
 } from '@mui/material';
-import { Event, Explore, Home, Map, Menu, People } from '@mui/icons-material/';
+import { CSSObject, styled, Theme } from '@mui/material/styles';
+import { Event, Explore, Home, Map, People } from '@mui/icons-material/';
 
-import makeStyles from '@mui/styles/makeStyles';
+import messageIds from './l10n/messageIds';
+import OrganizationsDataModel from 'features/organizations/models/OrganizationsDataModel';
+import { useMessages } from 'core/i18n';
+import useModel from 'core/useModel';
+import ZUIFuture from './ZUIFuture';
 
-import ZUILogo from './ZUILogo';
+const drawerWidth = 240;
 
-const drawerWidth = '5rem';
-
-const useStyles = makeStyles((theme) => ({
-  appBar: {
-    [theme.breakpoints.up('sm')]: {
-      display: 'none',
-    },
-  },
-  content: {
-    flexGrow: 1,
-    padding: theme.spacing(3),
-  },
-  drawer: {
-    [theme.breakpoints.up('sm')]: {
-      flexShrink: 0,
-      width: drawerWidth,
-    },
-  },
-  drawerPaper: {
-    display: 'none',
-    width: drawerWidth,
-    [theme.breakpoints.up('sm')]: {
-      display: 'block',
-    },
-  },
-  menuButton: {
-    marginRight: theme.spacing(2),
-    [theme.breakpoints.up('sm')]: {
-      display: 'none',
-    },
-  },
-  root: {
-    display: 'flex',
-  },
-  roundButton: {
-    background: 'white',
-    borderRadius: '50%',
-    height: '3rem',
-    width: '3rem',
+const closedMixin = (theme: Theme): CSSObject => ({
+  overflowX: 'hidden',
+  transition: theme.transitions.create('width', {
+    duration: theme.transitions.duration.leavingScreen,
+    easing: theme.transitions.easing.sharp,
+  }),
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up('sm')]: {
+    width: `calc(${theme.spacing(8)} + 1px)`,
   },
+});
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  overflowX: 'hidden',
+  transition: theme.transitions.create('width', {
+    duration: theme.transitions.duration.enteringScreen,
+    easing: theme.transitions.easing.sharp,
+  }),
+  width: drawerWidth,
+});
+
+const Drawer = styled(MuiDrawer, {
+  shouldForwardProp: (prop) => prop !== 'open',
+})(({ theme, open }) => ({
+  boxSizing: 'border-box',
+  flexShrink: 0,
+  ...(open && {
+    ...openedMixin(theme),
+    '& .MuiDrawer-paper': openedMixin(theme),
+  }),
+  ...(!open && {
+    ...closedMixin(theme),
+    '& .MuiDrawer-paper': closedMixin(theme),
+  }),
+  whiteSpace: 'nowrap',
+  width: drawerWidth,
+}));
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'flex-start',
+  padding: theme.spacing(0, 1),
```

Comment:
Why the move away from `makeStyles()` to `styled()`? I don't believe we use `styled()` anywhere else in the project and I'm hesitant to introduce new patterns.